pkg/model/context.go

```diff
@@ -221,6 +221,8 @@ func (m *KopsModelContext) CloudTagsForInstanceGroup(ig *kops.InstanceGroup) (ma
 		labels[awstasks.CloudTagInstanceGroupRolePrefix+strings.ToLower(string(kops.InstanceGroupRoleBastion))] = "1"
 	}
 
+	labels["kops.k8s.io/instancegroup" /*nodeidentityaws.CloudTagInstanceGroupName*/] = ig.Name
```

Comment:
is this valid syntax and should be like this? Otherwise this PR is lgtm

Reply:
Ah - it's awkward syntax but it's just an inline comment.  The constant value will be extracted in the follow on PR, but that nodeidentityaws package doesn't exist yet.